src/RuleProvider/DefaultRuleProvider.php

```diff
@@ -798,6 +1096,39 @@ class DefaultRuleProvider implements RuleProviderInterface
     'ū' => 'u',
     'ž' => 'z',
   ),
+  'french' => 
+  array (
+    'À' => 'A',
+    'Â' => 'A',
+    'Æ' => 'AE',
+    'Ç' => 'C',
+    'É' => 'E',
+    'È' => 'E',
+    'Ê' => 'E',
+    'Ë' => 'E',
+    'Ï' => 'I',
+    'Î' => 'I',
+    'Ô' => 'O',
+    'Œ' => 'OE',
+    'Ù' => 'U',
+    'Û' => 'U',
+    'Ü' => 'U',
+    'à' => 'a',
+    'â' => 'a',
+    'æ' => 'ae',
+    'ç' => 'c',
+    'é' => 'e',
+    'è' => 'e',
+    'ê' => 'e',
+    'ë' => 'e',
+    'ï' => 'i',
+    'î' => 'i',
+    'ô' => 'o',
+    'œ' => 'oe',
+    'ù' => 'u',
+    'û' => 'u',
+    'ü' => 'u',
+  ),
```

Comment:
Two characters are missing in the French rulset: ÿ and Ÿ.
These characters are not commonly used (e.g. some city names or last names) but should availble in any case.

See https://fr.wikipedia.org/wiki/%C5%B8 for more information.